_rules/error-field-is-not-identified-using-color-only

```diff
@@ -0,0 +1,107 @@
+---
+id:
+name: Error field is not identified using color differences only
+rule_type: atomic
+description: |
+  This rule checks that error fields can be identified not only by color differences but through another visual means.
+accessibility_requirements: # Remove whatever is not applicable
+  wcag21:1.4.1: # Use of Color (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  -  DOM Tree
+  -  CSS Styling
+acknowledgments:
+  authors:
+    -  Giacomo Petri
+---
+
+## Applicability
+
+This rule applies to any error field that is [visible][].
+
+## Expectation (1)
+
+Each test target, together with its visual context, identify the presence of errors not only through color differences.
```

Comment:
What does "not only through color differences" mean?
Notably, does color refers to the full RGB color or only to the hue? Is a difference in contrast good enough (from which treshold).

Look at #1010 to see some other discussions around 1.4.1 and what constitute a good enough difference, OTOH there might be other thing this rule needs to consider.

Reply:
The idea was to follow the WCAG 1.4.1 requirement

> If content is conveyed through the use of colors that differ not only in their hue, but that also have a significant difference in lightness, then this counts as an additional visual distinction, as long as the difference in relative luminance between the colors leads to a contrast ratio of 3:1 or greater.

So color differences was intended as differences only in their hue, as also specified in Assumptions section. Per other feedbacks I've moved assumption notes in expectation.